core/tests/karma.conf.js

```diff
@@ -1,32 +1,26 @@
+var generatedJs;
+process.argv.forEach(function(argument) {
+  if (argument == 'minify') {
+    generatedJs = 'third_party/generated/prod/js/third_party.js';
+  } else {
```

Comment:
Given that it's a list of strings, should we do 

    generatedJs = ...dev...

in line 1, and do

     process.argv.some(function(...) {});

to handle the prod case? There the logic is: if some argument is "minify" (or, possibly, "--minify", to be in sync with the CLI at http://karma-runner.github.io/0.13/config/configuration-file.html), we change the path to prod. Otherwise, I'm concerned that if we ever have additional args after the "minify" one, we may set it back to dev by accident.